languages/elixir/exercises/concept/stack-underflow/.meta/config.json

```diff
@@ -13,7 +13,7 @@
   ],
   "editor": {
     "solution_files": ["lib/rpn_calculator.ex"],
-    "test_files": ["test/rpn_calculator_test.exs"]
+    "test_files": ["test/rpn_calculator/exception_test.exs"]
```

Comment:
Oops! I think we should always have a matching `test` and `lib` paths, it would be nice to change either one.

Reply:
Agreed. I moved the implementation files into subdirectories to match elixir convention. See #2677